dialect/sql/builder_test.go

```diff
@@ -1063,6 +1063,50 @@ func TestBuilder(t *testing.T) {
 			wantQuery: `SELECT * FROM "groups" JOIN (SELECT "user_groups"."id" FROM "user_groups" JOIN "users" AS "t0" ON "user_groups"."id" = "t0"."id2" WHERE "t0"."id" = $1) AS "t1" ON "groups"."id" = "t1"."id" LIMIT $2`,
 			wantArgs:  []interface{}{"baz", 1},
 		},
+		{
+			input: func() Querier {
+				t1 := Table("users")
+				return Dialect(dialect.Postgres).
+					Select().
+					From(t1).
+					Where(CompositeGT(t1.Columns("id", "name"), 1, "Ariel"))
+			}(),
+			wantQuery: `SELECT * FROM "users" WHERE ("users"."id", "users"."name") > ($1, $2)`,
```

Comment:
@chris-rock, I'm not familiar with this syntax - `(columns..ץ) OP (args...)`.
Would you mind add here a reference to docs, or explain how does it work (or what does it mean)? Also, the support in the different dialects (SQLite, MySQL and PostgreSQL).

Reply:
The inspiration for the implementation came from https://use-the-index-luke.com/no-offset. I have tested the syntax in Sqlite and Postgres. [SQLite Example](https://www.sqlite.org/rowvalue.html#scrolling_window_queries):
```
SELECT * FROM contacts
 WHERE (lastname,firstname) > (?1,?2)
 ORDER BY lastname, firstname
 LIMIT 7;
```

Reply:
The syntax is available for a long time, another good example: https://www.jooq.org/doc/3.12/manual/sql-building/sql-statements/select-statement/seek-clause/